src/components/BalancesList.tsx

```diff
@@ -56,6 +62,8 @@ interface BalancesListProps {
 export const BalancesList: React.FC<BalancesListProps> = ({
   publicKey,
   network,
+  showTitleIcon = false,
```

Comment:
I wonder if we should just always show the icon for UI consistency, I've dropped a question for Charles here: https://www.figma.com/design/KwkHXQxbNmDllwermJtnRu?node-id=3707-84986#1238601086